man/org.freedesktop.systemd1.xml

```diff
@@ -4603,10 +4603,10 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       <varname>ExtensionDirectories</varname>
       see systemd.exec(5) for their meaning.</para>
 
-      <para><varname>MemoryAvailable</varname> indicates how much unused memory is available to the unit before
-      the <literal>MemoryMax</literal> or <literal>MemoryHigh</literal> (whichever is lower) limit set by the cgroup
-      memory controller is reached. It will take into consideration limits on all parent slices, other than the
-      limits set on the unit itself.</para>
+      <para><varname>MemoryAvailable</varname> is a trick you can pull to impress people by predicting bright future.
```

Comment:
I don't like this rewording, when cg2 with memory accounting is enabled it works well, please drop the change and instead mention that memory accounting needs to be enabled, and how physical memory is used

Reply:
I admit it was biased. Reworded more factually.